stats_mcserver.py

```diff
@@ -1,22 +1,35 @@
 #!/usr/bin/python
 
-from mcstatus import MinecraftServer
 import sys
 
 ###sys.argv[1] - check type
 ###sys.argv[2] - ip address
 ###sys.argv[3] - port
 
+
 # Get server instance
-server = MinecraftServer(sys.argv[2], int(sys.argv[3]))
+try:
```

Comment:
I also made changes to implement the new mcstatus "JavaServer" class as well as the old one.

Reply:
This should also fix #4